src/app/clusters/on-off-server/on-off-server.h

```diff
@@ -96,19 +96,20 @@ struct OnOffEffect
     chip::EndpointId mEndpoint;
     OffWithEffectTriggerCommand mOffWithEffectTrigger = nullptr;
     chip::app::Clusters::OnOff::OnOffEffectIdentifier mEffectIdentifier;
-    uint8_t mEffectVariant;
+    chip::app::Clusters::OnOff::OnOffDelayedAllOffEffectVariant mEffectVariant;
```

Comment:
How does this work if the effect identifier is DyingLight?  It seems to me like this needs to stay a uint8_t....